packages/neo-one-client-common/src/helpers.ts

```diff
@@ -8,102 +8,113 @@ import { AddressString, PrivateKeyString, PublicKeyString } from './types';
  * @param publicKey hex-encoded public key
```

Comment:
Pull these changes out into a separate PR to keep the changelog neat. (I think it should be `refactor` as the type of change)